docs/generate_docs.py

```diff
@@ -278,6 +294,8 @@ def create_tutorial_notebooks(tutorial):
         os.path.basename(os.getcwd()) == "docs"
     ), "Only run this utility from inside the docs/ directory!"
 
+    import nbformat as nbf
```

Comment:
Why was this removed from the dev requirements and then used here?

Reply:
This helper function isn't actually used anywhere right now; so, in an effort to reduce our dependencies I removed it as a "required" dev dependency and tucked the import inside this unused function to avoid `ImportError`s.